features/pull_request.feature

```diff
@@ -538,6 +538,42 @@ Feature: hub pull-request
     Then the output should contain exactly "https://github.com/mislav/coral/pull/12\n"
     And the file ".git/PULLREQ_EDITMSG" should not exist
 
+  Scenario: Title and body from commit
```

Comment:
Can you please rename this scenario "Title and body from commit when PR template exists" and create a copy of this scenario, but a simpler version of it where the PR template doesn't exist in the first place and the message is simply taken from the commit message? Thank you!

Reply:
Done :)

FYI it seems that CI tests are actually non-deterministic. I've encountered random failed test of untouched part of the codebase which were resolved in another run - see https://travis-ci.org/github/hub/builds/574154296 and https://travis-ci.org/github/hub/builds/574155640 (the code was exactly the same, I've just amend commit msg & push to trigger CI)